openmls/src/schedule/mod.rs

```diff
@@ -208,6 +211,19 @@ impl From<Secret> for InitSecret {
     }
 }
 
+/// Creates a string from the given MLS `ProtocolVersion` for the computation of
+/// the `init_secret` when creating or processing a commit with an external init
+/// proposal.
+fn hpke_info_from_version(version: ProtocolVersion) -> String {
+    match version {
```

Comment:
These are all static strings. No need to make this more expensive than necessary. You can return a `&'static str` here.

Reply:
I did the concatenation manually for now. I think there is some macro magic that I could use to avoid appending the "external init" to each version, but I thought that was a bit overkill. Also, considering https://github.com/mlswg/mls-protocol/issues/531, we're probably going to touch this again soon anyway.